app/controllers/catalog_controller.rb

```diff
@@ -27,12 +27,13 @@ def self.modified_field
     solr_name('date_modified', :stored_sortable, type: :date)
   end
 
-  configure_blacklight do |config|          config.view.gallery.partials = [:index_header, :index]
-          config.view.masonry.partials = [:index]
-          config.view.slideshow.partials = [:index]
+  configure_blacklight do |config|
+    config.view.gallery.partials = [:index_header, :index]
+    config.view.masonry.partials = [:index]
+    config.view.slideshow.partials = [:index]
 
-          config.show.tile_source_field = :content_metadata_image_iiif_info_ssm
-          config.show.partials.insert(1, :openseadragon)
+    config.show.tile_source_field = :content_metadata_image_iiif_info_ssm
+    config.show.partials.insert(1, :openseadragon)
 
     #Show gallery view
     config.view.gallery.partials = [:index_header, :index]
```

Comment:
Lines 38 and 39 seem to be repeats of 31 and 32. Are they supposed to be config.show instead of config.view?